cfgov/unprocessed/apps/mmt-cost-of-credit-tool/js/views/tool/index.js

```diff
@@ -0,0 +1,132 @@
+import { Link } from "react-router-dom";
+
+import { TotalPurchaseAmount } from "../../components/total-purchase-amount";
+import { DownPayment } from "../../components/down-payment";
+import { CreditApr } from "../../components/credit-apr";
+import { MonthlyPayment } from "../../components/monthly-payment";
+// import { DealsAndPromotions } from "../../components/deals-and-promotions";
+import { DealOrPromo } from "../../components/deal-or-promo";
+import { TrueCostBreakdown } from "../../components/true-cost-breakdown";
+import { AlternatePayoffScenarios } from "../../components/alternate-payoff-scenarios";
+import { TipsAndResources } from "../../components/tips-and-resources";
+
+// import { NonPaymentConsequences } from "../../components/non-payment-consequences";
+// import { Habits } from "../../components/habits";
+// import { Decision } from "../../components/decision";
+// import { SectionTip } from "../../components/section-tip";
+// import { RadioButtonQuestion } from "../../components/controls/radio-button-question";
+// import { DollarSliderInput } from "../../components/controls/dollar-slider-input";
+// import { PercentSliderInput } from "../../components/controls/percent-slider-input";
+// import { MonthSliderInput } from "../../components/controls/month-slider-input";
+// import { DollarInput } from "../../components/controls/dollar-input";
+// import { PercentForMonthsInput } from "../../components/controls/percent-for-months-input";
+
+export default function Tool() {
+  return (
+    <section id="tool">
+      <div className="m-tool__tag">True Cost Tool</div>
+      <h1>Enter the purchase details</h1>
+      <div className="m-tool-card__desc">
+        It's okay to estimate if you're not sure.
+      </div>
+      <div className="o-tool-card-group">
+        <div className="m-tool-card">
+          {" "}
```

Comment:
What's with all these space literals?

Reply:
I don't know.  Something in VS Code keeps adding them.  I removed them.

Reply:
This is related to your auto-formatting setting. Make sure every computer you're working on this codebase on has that "Auto-format on save" setting checkbox unchecked. We only want to run the code through Prettier deliberately.